app/views/_create-secret.html

```diff
@@ -0,0 +1,267 @@
+<!-- Error messages -->
+<parse-error error="error" ng-if="error"></parse-error>
```

Comment:
The `parse-error` directive was meant specifically for YAML and JSON parsing errors. Is there a reason not to use the `alerts` directive here inside the dialog?